dali/pipeline/executor/lowered_graph.cc

```diff
@@ -351,6 +357,7 @@ void OpGraph::RemoveOpNode(OpNodeId id) {
     Node(parent_id).children.erase(op_nodes_.back().id);
   }
   // assume that we removed one element
```

Comment:
Why "assume"? If we're not sure, maybe we should have an `assert` for this?

Reply:
That's not my comment, is it? I'll check if I can easily assert on the result of erase, but even then `assert` is a bit problematic, because in Release build `assert(x.erase())` will remove the call to `erase` and `bool erased = erase()` will complain about unused variable :\

Reply:
OK, after looking at the code I'm inclined towards just removing this comment. There's nothing to assume there. We _removed_ one element, there's nothing to assume here.

Reply:
Great, thanks!